src/products/mozilla_org/testing.md

```diff
@@ -44,7 +44,7 @@ You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.or
 * Click the download link, you should be able to download the product in your language, if it is localized, such as Firefox.
 * Font support and readability.
 * Footer: verify that the translation of the link is coherent and the link is functional.
-* Language list: is your language listed as one of the options? Check https://www-dev.allizom.org/en-US to confirm.
+* Language list: is your language listed as one of the options? Check https://www-dev.allizom.org/firefox/all/ to confirm. See [detailed steps below](https://github.com/mozilla-l10n/localizer-documentation/blob/main/src/products/mozilla_org/testing.md#firefoxallftl) to get to the language list.
```

Comment:
You don't need the full URL to link within a page, just the part `#firefoxallftl`.